src/libmongoc/tests/test-mongoc-scram.c

```diff
@@ -669,6 +685,49 @@ test_mongoc_saslprep_auth_no_icu (void *ctx)
    _drop_saslprep_users ();
 }
 
+static void
+test_mongoc_utf8_char_length (void)
+{
+   const char *c0 = ",";
+   const char *c1 = "ɶ";
+   const char *c2 = "ྡྷ";
+   const char *c3 = "🌂";
+
+   ASSERT_CMPINT (_mongoc_utf8_char_length (c0), ==, 1);
```

Comment:
```suggestion
   ASSERT_CMPSIZE_T (_mongoc_utf8_char_length (c0), ==, 1u);
```
Use `size_t` to represent lengths.